Makefile

```diff
@@ -1,12 +1,21 @@
 SHELL = /usr/bin/env bash
```

Comment:
Can you change `make test` like this?: https://github.com/fyntex/lib-cl-sii-python/blob/73bc445483a3834dc8491d5c5394414248e4d9f4/Makefile#L80-L81